test_unstructured_ingest/check-num-dirs-output.sh

```diff
@@ -16,7 +16,7 @@ OUTPUT_DIR=$SCRIPT_DIR/structured-output/$OUTPUT_FOLDER_NAME
 
 NUMBER_OF_FOUND_DIRS="$(find "$OUTPUT_DIR" -type d -exec printf '.' \; | wc -c | xargs)"
 
-if [ "$NUMBER_OF_FOUND_DIRS" != "$EXPECTED_NUM_DIRS" ]; then
+if [ "$NUMBER_OF_FOUND_DIRS" -ne "$EXPECTED_NUM_DIRS" ]; then
```

Comment:
worth adding a NOTE before each condition with a warning not to change to double-bracket syntax due to CI issues?

Reply:
yep, done!